### ISSUE TITLE
Client side 404 error page

### ISSUE THREAD

**Issue by intelagense** [Together-100Devs/Together#452]
## Description
Mistyped URLs currently lead to a blank page on the front end, which isn't user-friendly. Let's add a 404 error page with links back to the home page, calendar view, and GitHub.

## Are there any potential issues you foresee with this feature
The overall theme should match the current design and include the logo and information about the error. 

## Related issues
Ironically enough, this is related to issue #404 (Convert Together to React Router) Which was closed when #423 was merged.

## Additional context
This should be somewhat built into React Router already. 

We can introduce some subtle humor on this page to ease the frustration associated with encountering an error. However, it's important to keep in mind that our user base includes individuals who may not be fluent in English or come from diverse cultural backgrounds. 

_If you're interested in working on this feature, please comment so you can be assigned_

**Comment by stanleynzegwu** [comment]
Hello,

I'm interested in working on this feature, and I'm excited to note that this will be my first contribution here. While I'm waiting for the green flag to get started, let me try and get the project running on my local machine.

I believe creating a 404 error page with a user-friendly design is important for improving the user experience. I'd be happy to contribute to this and ensure that it aligns with the current design and includes the necessary elements like the logo and error information.

Please assign this task to me, and I'll get started on it.

**Comment by intelagense** [comment]
@stanleynzegwu Hey, sorry, I am just now seeing this. Do you still want to work on the issue?

**Comment by MelCode1013** [comment]
@intelagense 
Hi! If @stanleynzegwu hasn't picked up this issue, I would love to work on it!

**Comment by stanleynzegwu** [comment]
> @stanleynzegwu Hey, sorry, I am just now seeing this. Do you still want to work on the issue?

I've been waiting for the green light to proceed, but it didn't come. I started another project and didn't realize you had replied. I just saw @MelCode1013's comment in my email, which prompted me to check now.

 Hi, yes, I still want to work on the issue

**Comment by intelagense** [comment]
Y'all could also maybe work on it "Together?" 😆 

Might be a little slow response rate on the project this time of year. 🙏

Feel free to sent a note in the Discord thread in general as well if you need to get in contact.

**Comment by vguzman812** [comment]
If this hasn't been done yet, I have a solution ready for a PR.

**Comment by matthiasjrichter** [comment]
https://github.com/Ashir-Valjee and I will work on this. Expect a progress report in the next meetup.

**Comment by Ashir-Valjee** [comment]
Hi happy to do this
